packages/components/src/popover/index.js

```diff
@@ -94,6 +96,47 @@ class Popover extends Component {
 		this.toggleAutoRefresh( false );
 	}
 
+	/**
+	 * Shims an onFocusOutside callback to be compatible with a deprecated
+	 * onClickOutside prop function, if provided.
+	 *
+	 * @param {FocusEvent} event Focus event from onFocusOutside.
+	 */
+	onFocusOutside( event ) {
```

Comment:
How can I test this shim? By updating one of the places with usage?

Reply:
You could locally revert one or both of 55ca76a5427876ab91e6966272e973fe295e0275 or 3e8ffc4673db9f9527173d03b5018b53d7c78a9d to restore those components' use of `onClickOutside`.